### ISSUE TITLE
Adjust font size

### ISSUE THREAD

**Issue by laserstrike** [standardnotes/mobile#31]
Possible for a future release?

**Comment by mobitar** [comment]
Yes, possible with themes in a future release for sure. Right now themes are mostly limited to colors but I plan on expanding this.

**Comment by Fohroer** [comment]
@mobitar when this function will be added?

**Comment by mortezadadgar** [comment]
Is there any chance we can get this feature

**Comment by JaspalSuri** [comment]
Hi @mortezadadgar, currently this isn't possible with themes or through the app's settings. You'll need to use the settings provided by the operating system running on your device (such as iOS or Android) to increase the font size of all (or some) apps. 

[Here are the instructions for iOS users](https://support.apple.com/guide/iphone/display-text-size-iph3e2e1fb0/ios) on increasing the size of text across all apps. [Here are instructions](https://lifehacker.com/how-to-set-unique-text-sizes-for-different-apps-in-ios-1847355175) on how to set the text size for certain apps on iOS15.

[Here are instructions](https://support.google.com/accessibility/android/answer/11183305?hl=en&visit_id=637691891946663328-2135810101&rd=1) for (some) Android users on changing the display and font size.
